src/presentation-3/normalize.ts

```diff
@@ -85,6 +86,68 @@ function mapToEntities(entities: Record<string, Record<string, NormalizedEntity>
   };
 }
 
+function merge(existing: any, incoming: any): any {
+  if (incoming) {
```

Comment:
Should this be:
```suggestion
  if (!incoming) {
```
To pick up the `false-y` value?

Reply:
omg, yes 🙀  thank you, shows how thoroughly I tested this 🙈